code-examples/python.py

```diff
@@ -1,45 +1,72 @@
-from hashlib import sha1
-from time import time
-import base64
-import urllib2
-import urllib
-import json
-import hmac
-
 """
 author: Evan Duffield <eduffield@iacquire.com>
-
 Thank you to iAcquire for sponsoring development of this module.
+
+Updated by Bryan Minor, Ph.D.
+--> Now works for Python 3
+Supported by envision.ai
 """
 
-follower_wonk_access_id = ""
-follower_wonk_secret_key = ""
+# External libraries
+from hashlib import sha1
+from time import time
+import base64
+import urllib.request
```

Comment:
These are exclusively the Python 3 names for these imports. It would be better to support both Python 2 and 3 with the same code rather than replacing the Python 2 version with a Python 3-only version.